_docs/2.0.0/usage/menas-conformance-rules.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Usage - Menas Conformance Rules
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+## Table Of Contents
+<!-- toc -->
+- [Table Of Contents](#table-of-contents)
+- [Intro](#intro)
+- [Casting Conformance Rule](#casting-conformance-rule)
+- [Concatenation Conformance Rule](#concatenation-conformance-rule)
+- [Drop Conformance Rule](#drop-conformance-rule)
+- [Literal Conformance Rule](#literal-conformance-rule)
+- [Mapping Conformance Rule](#mapping-conformance-rule)
+- [Negation Conformance Rule](#negation-conformance-rule)
+- [SingleColumn Conformance Rule](#singlecolumn-conformance-rule)
+- [SparkSessionConf Conformance Rule](#sparksessionconf-conformance-rule)
+- [Uppercase Conformance Rule](#uppercase-conformance-rule)
+- [FillNulls Conformance Rule](#fillnulls-conformance-rule)
+<!-- tocstop -->
+
+## Intro
+
+Conformance Rules are a way for the user to enhance the data. Every conformance rule has an output column and an option for running a control measure. Output column defines the name of the output column into which the rule will output. Control measure check will run an Atum control measure check as in previouse stages as defined in controls of `_INFO` file.
+
+Every column also has one or more input columns.
+
+**Important** - We never override a column. Each rule produces a new column.
+
+## Casting Conformance Rule
+
+Allows users to cast a specific column to another type. This conformance rules provides a selection of other types.
+
+## Concatenation Conformance Rule
+
+Concatenation rule concatenates multiple input columns together into a single column. Columns are first transformed into their string representation.
+
+## Drop Conformance Rule
+
+Drop Conformance rule remove a column from the DataFrame.
+
+## Literal Conformance Rule
+
+Literal rule add a column with a string literal provided by the user.
+
+## Mapping Conformance Rule
+
+To use a mapping conformance rule, user first needs to define a Mapping Table in menas. Mapping Tables have the same properties and rules around them as Datasets, plus it is expected of them to be in a parquet format.
+
+When defining a mapping conformance rule, users first needs to pick a correct Mapping Table and correct version. Then there is a question if the data can have Null values in join conditions. This means if the join that will be executed should be null safe or not.
+
+Then the join conditions. A join condition condition conveys a relationship between the Dataset and MappingTable. The join condition specifies how the rows from one table will be combined with the rows of another table. This is based on the equality of the rows.
```

Comment:
```suggestion
Following, a join condition condition conveys a relationship between the table data and Mapping Table. The join condition specifies how the rows from one table will be combined with the rows of the other table. This is based on the equality of the rows.
```

Reply:
- I don't like this following part because it is jumping to the next section without tellin user we are moving to the next section
- I don't like the table data. We have never spoken about table data, we always speak about Dataset

Reply:
Dataset is good, I think. Would make it `dataset`, I don't see the reason to capitalize it.

Reply:
I think the _Following_ signals it's another section. But what about:
```suggestion
Then the join conditions conveys the relationship between the Dataset and Mapping Table. The join condition specifies how the rows from one table will be combined with the rows of another table. This is based on the equality of the values in the selected columns.
```
Sorry, changed the end too, not the whole rows are relevant.